### PR TITLE
chore: add ownership for aws propagator

### PR DESCRIPTION
**Pull request by dyladan**
First official component owners @willarmiros and @NathanielRN

### CODE REVIEW

File: .github/component_owners.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Test configuration
+
+components:
```
Comment (willarmiros) [inline .github/component_owners.yml:3]: Shall we also own https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/packages/opentelemetry-id-generator-aws-xray?

Comment (willarmiros) [reply .github/component_owners.yml:3]: And https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-aws-lambda?

**Review by obecny — APPROVED** [review]
lgtm

File: .github/component_owners.yml (hunk 1 above)
Comment (dyladan) [reply .github/component_owners.yml:3]: yes

**Review by willarmiros — APPROVED** [review]
🎉

**Pull request merged**